addons/web/static/src/views/graph/graph_view.scss

```diff
@@ -3,6 +3,12 @@
     background-color: $o-view-background-color;
 }
 
+.o_graph_view {
+    .o_cp_bottom_left {
+        display: block;
```

Comment:
Remainder: Remove this styles  in FW port to master and move both `btn-group` in pivot&graph view in a single div.